### PR TITLE
Add Fremennik Exiles monsters to the NPC health list

### PR DESCRIPTION
**Pull request by huddy987**
Got the levels and health values from the wiki.

### CODE REVIEW

**Comment by huddy987** [comment]
Third commit is due to duplicate Basilisk_61 entries in the dictionary

File: runelite-client/src/main/resources/npc_health.json [hunk 1]
```diff
@@ -1160,5 +1160,11 @@
 	"Alchemical Hydra_426": 1100,
 	"Undead Druid_105": 140,
 	"Temple Spider_75": 70,
-	"Sarachnis_318": 400
+	"Sarachnis_318": 400,
+	"Basilisk Youngling_57" : 60,
+	"Monstrous Basilisk_135" : 170,
+	"Typhor_194" : 280,
+	"The Jormungand_363" : 600,
+	"Basilisk Knight_182" : 200,
```
Comment (Alexsuperfly) [inline runelite-client/src/main/resources/npc_health.json]: after the update today they now have 300hp

Comment (huddy987) [reply runelite-client/src/main/resources/npc_health.json]: Fixed, Should be good now.

File: runelite-client/src/main/resources/npc_health.json [hunk 2]
```diff
@@ -1160,5 +1160,11 @@
...
 	"Undead Druid_105": 140,
 	"Temple Spider_75": 70,
-	"Sarachnis_318": 400
+	"Sarachnis_318": 400,
+	"Basilisk Youngling_57" : 60,
+	"Monstrous Basilisk_135" : 170,
+	"Typhor_194" : 280,
+	"The Jormungand_363" : 600,
+	"Basilisk Knight_204" : 300,
+	"Basilisk Sentinel_351" : 460
```
Comment (Adam-) [inline runelite-client/src/main/resources/npc_health.json]: I don't think this is right https://oldschool.runescape.wiki/w/Basilisk_Sentinel

Comment (huddy987) [reply runelite-client/src/main/resources/npc_health.json]: It looks like he was buffed since my commit. I'll fix this.

**Pull request merged**